.github/workflows/pr.ci.yml

```diff
@@ -56,13 +56,8 @@ jobs:
           mkdir -p priv/plts
           mix dialyzer --plt
 
-        # Replace <code-extensions> with file extensions that should trigger this check. Replace with .* to allow anything.
-      - name: Run exercism/elixir ci pre-check (stub files, config integrity) for changed exercises
-        run: |
-          PULL_REQUEST_URL=$(jq -r ".pull_request.url" "$GITHUB_EVENT_PATH")
-          curl --url $"${PULL_REQUEST_URL}/files" --header 'authorization: Bearer ${{ secrets.GITHUB_TOKEN }}' | \
-            jq -c '.[] | select(.status == "added" or .status == "modified") | select(.filename | match("\\.(ex|exs|md|json)$")) | .filename' | \
-            xargs -r bin/pr-check.sh
```

Comment:
IMO this check doesn't make much sense because the scripts `bin/pr-check.sh` and `bin/pr.sh` don't have an option to only work for selected exercises or files, they always check all of the files when run.

Right now, `bin/pr-check.sh` and `bin/pr.sh` won't run if no `ex|exs|md|json` were changed, which means they won't run if the scripts themselves were changed! Like in the first commit in this PR.

I think the most reasonable and safe choice that doesn't require a lot of extra work is to always run all checks.